src/RoleObjects/SliderData.js

```diff
@@ -1,115 +1,190 @@
+import KeyCodes from 'keycodes-enum';
 import _ from 'lodash';
 import InputTagData from './InputTagData';
 
 /**
  * Class for the slider role
  */
-
```

Comment:
There was a bunch of cleaning up comment formatting in this file since I was in it anyways